test/functional/services/dashboard/panel_actions.ts

```diff
@@ -190,11 +191,14 @@ export class DashboardPanelActionsService extends FtrService {
 
   async unlinkFromLibary(parent?: WebElementWrapper) {
     this.log.debug('unlinkFromLibrary');
-    const libraryNotification = parent
-      ? await this.testSubjects.findDescendant(LIBRARY_NOTIFICATION_TEST_SUBJ, parent)
-      : await this.testSubjects.find(LIBRARY_NOTIFICATION_TEST_SUBJ);
-    await libraryNotification.click();
-    await this.testSubjects.click('libraryNotificationUnlinkButton');
+    await this.retry.tryForTime(30000, async () => {
```

Comment:
I'm wondering if there's a way we can make this type of workflow more transparent..I'm guessing it's not clear on first implementation that this needs to be done.

Don't want to hold up the changes here.